imagery/i.group/i.group.md

```diff
@@ -29,6 +29,58 @@ only the visible light bands of Landsat-7:
 i.group group=vis_bands subgroup=vis_bands input=lsat7_2000_10,lsat7_2000_20,lsat7_2000_30
 ```
 
+To list files in the `vis_bands` subgroup under the `vis_bands` group:
```

Comment:
Could you separate the -l and -s examples and use the content tabs? I am thinking of using the plain format and python with json